### PR TITLE
rtl872xd: improve lwip options to fix http test faults

### PR DESCRIPTION
**Pull request by jinliangli**
[Detail]
improve lwip options to fix http test faults

[Verified Cases]
Build Pass: <py_engine_demo>
Test Pass:  <py_engine_demo>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
auto build success,number:212

**Comment by skylarCai** [comment]
autotest fail,number:1571

**Review by yanxiaoyong-ch — COMMENTED** [review]
this looks ok for me

File: hardware/chip/rtl872xd/arch/lwipopts.h [hunk 1]
```diff
@@ -46,11 +46,10 @@
```
Comment (skylarCai) [inline hardware/chip/rtl872xd/arch/lwipopts.h:46]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**